.github/workflows/ci-cd-payment-service.yml

```
@@ -34,7 +34,9 @@ jobs:
         distribution: 'temurin'
 
     - name: Set up application.properties
-      run: echo "${{ env.APPLICATION }}" > "${{ env.SERVICE_NAME }}/src/main/resources/application.properties"
+      run : |
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)